promql/fuzz.go

```diff
@@ -0,0 +1,67 @@
+package promql
+// +build gofuzz
```

Comment:
Couple of style nits:

* For consistency with other Prometheus components, let's do this order:
  * license header comment (see e.g. https://github.com/prometheus/prometheus/blob/master/cmd/prometheus/main.go#L1-L12)
  * build tag comment
  * package definition
* We don't use `/* ... */`-style comments, so please switch to `//` ones.